src/FhirLoader.Common/ProcessedBundle.cs

```diff
@@ -11,5 +11,7 @@ public class ProcessedBundle
         public string? BundleText;
         public int BundleCount;
         public string? BundleFileName;
+        public string? ResourceType;
```

Comment:
Since this isn't always a bundle, can we change this to "ProcessedResource"? Or add a new class for Resource so it's clear when we are using a bundle or not?

Reply:
Changed to ProcessedResource.